x/coinswap/simulation/operations.go

```diff
@@ -174,7 +174,7 @@ func SimulateMsgAddLiquidity(k keeper.Keeper, ak types.AccountKeeper, bk types.B
 			}
 		}
 
-		txGen := simappparams.MakeTestEncodingConfig().TxConfig
+		txGen := moduletestutil.MakeTestEncodingConfig().TxConfig
```

Comment:
Have there been any issues arising from the `TestEncodingConfig` not registering the interfaces of custom modules like coinswap? Please check if there are any problems, and if so, it would be good to refactor [this code](https://github.com/b-harvest/Canto/blob/676ff7253e417372e5c929dd5ace181f97a92713/cmd/cantod/root.go#L57-L73) to use the same encoding config for tests and simulations as well.

Reply:
Using TestEncodingConfig, even if we do not pass modules, the msg will be recognized and reach the msg server.
`gogoProtoRegistry.FindDescriptorByName(name)` already contains the custom module's proto. descriptor.